card-unification.md

```diff
@@ -0,0 +1,90 @@
+# Card Unification
+
+  * [ ] Change card shape to object with `type`, `name`, `render` and optional `edit` properties
+  * [ ] Card's `type` is validated by renderer (dom renderer cannot render 'text', e.g.)
+  * [ ] Change arguments passed by editor to card's `render` (or `edit`) method
+    * [ ] single argument object with `name`, `buffer`, `isInEditor`, `options`, `editorEnv` and `payload` properties
+  * [ ] Return value of card's `render` (or `edit`) method is appended/concatenated by renderer
+  * [ ] card can register teardown callback by calling `buffer.onTeardown(teardownFn)`
+  * [ ] Change editor-dom renderer to clear child elements from card element on teardown
+  * [ ] Renderer `constructor` signature changes to accept cards (array), cardOptions (object) arguments
```

Comment:
cards, atoms, cardOptions

What happens to `editor` and `unknownCardHandler`? Is this the `renderer` here: https://github.com/bustlelabs/mobiledoc-kit/blob/master/src/js/renderers/editor-dom.js#L360?